src/Core/src/Platform/Android/Resources/values/styles.xml

```diff
@@ -65,6 +65,8 @@
       <!-- remove all the min sizes as MAUI manages it -->
       <item name="android:minWidth">0dp</item>
       <item name="android:minHeight">0dp</item>
+      <!-- remove the button tint as MAUI manages it -->
+      <item name="buttonTint">?attr/colorPrimary</item>
```

Comment:
It can also be set to `@null`

Reply:
I think `colorPrimary` is defined in the templates only? What will happen if you were to remove it?

Maybe `@null` or anything else may be better? Of does this color always exist in a plain android app with no styles or colors?